### ISSUE TITLE
I wonder if this application could be runned in PC?

### ISSUE THREAD

**Issue by ByteChen** [hairu/freelss#18]
I plan to build a simple 3D scaning system,and your code is very appreciated,so if I can use it in my system after some modifications,that would be of great help!

**Comment by hairu** [comment]
Yes it can but support would need to be extended to whatever camera technology is used (DirectShow, Video4Linux, etc) and you will need to have a way of programatically controlling the lasers and motor (an Arduino over USB could be used for this).
